app/uk/gov/hmrc/alcoholdutyreturns/controllers/actions/AuthorisedAction.scala

```diff
@@ -42,10 +44,18 @@ class BaseAuthorisedAction @Inject() (
     with BackendHeaderCarrierProvider
     with AuthorisedFunctions {
 
+  private val alcoholDutyEnrolment = "HMRC-AD-ORG"
```

Comment:
Add this to config as we do in the frontend please: https://github.com/hmrc/alcohol-duty-returns-frontend/blob/main/conf/application.conf#L107